automation_tools/__init__.py

```diff
@@ -3260,3 +3261,12 @@ def package_install(packages, sat_version=None):
     else:
         command = 'yum -y install {}'.format(packages)
     return command
+
+
+def setup_sendmail():
+    # there are some issues with mailing when using sendmail; workaround them:
+    # - with IPv6, mailing won't work => force postfix to use IPv4
```

Comment:
please rephrase the comment accordingly to my ltest findings: 
- without IPv6 address for lo interface mailing won't work